docs/source/installation/production/deb/apache.rst

```diff
@@ -379,11 +379,11 @@ Access ``https://YOURHOSTNAME`` in your browser and follow the steps
 displayed there to create your initial user.
 
 
-11. Install TeXLive
--------------------
+11. Set up PDF Document Generation
+----------------------------------
```

Comment:
I would not change this, because as far as the setup is concerned, TeXLive is what needs to be installed, the rest just works out of the box (except for weird fonts of course)